pkgs/tools/system/retry/default.nix

```
@@ -0,0 +1,33 @@
+{ lib, stdenv, fetchFromGitHub, libtool, automake, autoconf, which, txt2man }:
+stdenv.mkDerivation rec {
+    pname = "retry";
+    version = "1.04";
+
+    buildInputs = [ libtool automake autoconf which txt2man ];
+
+    src = fetchFromGitHub {
+        owner = "minfrin";
+        repo = "retry";
+        rev = "f6f1acd4b235e96dd9cb11263fb181ec44c3ece9";
```

```suggestion
        rev = "v${version}";
```
The commit pulling from should usually be the commit, under which the version was released. 
I haven't checked if they're the same commit here, but the version is always preferred.